src/main/java/frc/robot/commands/ToFIntakeCommand.java

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import frc.robot.subsystems.ArmSubsystem;
+import frc.robot.subsystems.IntakeSubsystem;
+import frc.robot.subsystems.ArmSubsystem.ArmState;
+import frc.robot.subsystems.IntakeSubsystem.FlightStates;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class ToFIntakeCommand extends CommandBase {
+  /** Creates a new ToFIntakeCommand. */
+  private final IntakeSubsystem intakeSubsystem;
+  private final ArmSubsystem armSubsystem;
+
+  public ToFIntakeCommand(IntakeSubsystem intakeSubsystem, ArmSubsystem armSubsystem) {
+    this.intakeSubsystem = intakeSubsystem;
+    this.armSubsystem = armSubsystem;
+    addRequirements(intakeSubsystem);
+    addRequirements(armSubsystem);
```

Comment:
Since this command only reads from the armSubsystem, addRequirements is not necessary and could cause a conflict with other arm commands